README.md

```diff
@@ -355,6 +305,37 @@ renderRichText(blok.richTextField, {
 
 ### Gatsby feature references
 
+### Prtial Hydration (RSC)
```

Comment:
Small typo here 👆

Reply:
Fixed! Thanks for spotting it @Dawntraoz 🙏 I'll commit it with the rest if there's more coming up from your review :)